Bigtable/src/ResumableStream.php

```diff
@@ -126,19 +126,22 @@ public function __construct(
      */
     public function readAll()
     {
-        $tries = 0;
-        $argumentFunction = $this->argumentFunction;
-        $retryFunction = $this->retryFunction;
+        $attempt = 0;
         do {
             $ex = null;
-            list($this->request, $this->callOptions) = $argumentFunction($this->request, $this->callOptions);
+            list($this->request, $this->callOptions) =
+                ($this->argumentFunction)($this->request, $this->callOptions);
 
             $completed = $this->pluck('requestCompleted', $this->callOptions, false);
 
             if ($completed !== true) {
+                // Send in "bigtable-attempt" header on each request
+                $headers = $this->callOptions['headers'] ?? [];
+                $headers['bigtable-attempt'] = [(string) ++$attempt];
```

Comment:
We'll do something like this instead, then
```suggestion
                if (++$attempt > 1) {
                    $headers['bigtable-attempt'] = [(string) $attempt];
                }
```